### ISSUE TITLE
@Facet on a package

### ISSUE THREAD

**Issue by sg-ad** [stoicflame/enunciate#233]
Hi!  I noticed that `@Facet` can be applied on a package.  I've tried it out but I'm not getting the result I'm expecting.  I wonder if either I'm doing it wrong, or my expectations are invalid.

Suppose I have class `com.project.billing.custom.acme.User` that is chock-full of jaxrs annotations.  In the package `com.project.billing.custom.acme` I create a `package-info.java` class with the following content:
```java
@Facet("acme")
package com.project.billing.custom.acme;

import com.webcohesion.enunciate.metadata.Facet;
```
and my configuration file contains the following:
```xml
<enunciate>
  ...
  <facets>
    <exclude name="acme"/>
  </facets>
</enunciate>
```
Am I right to expect that the `User` type will not appear in any of the generated documentation, including the XML schemas?

**Comment by stoicflame** [comment]
Is `User` referenced from any JAX-RS or JAX-WS endpoints?

**Comment by sg-ad** [comment]
Yes, but in another package.  The API declaration is like this:
```java
@Consumes({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
@Produces({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
public interface AccountApi {
  ...
  @GET
  @Path("/account/users/{userId}")
  public User readUser(@PathParam("userId") String userId);
```

**Comment by stoicflame** [comment]
So Enunciate assumes that it needs to pull in `User` in order to correctly document the `readUser` operation.

Is that an incorrect assumption? How did you expect Enunciate to document the response body of the `GET` method on the `/account/users/{userId}` resource?

**Comment by sg-ad** [comment]
I should've seen your point coming along like a ton of bricks.  :)  So in my attempt to extract a simple example out of our mass of code, I created an oversimplified one that sticks out like a sore thumb.  Nonetheless, I dug deeper into the problem that I was having and, indeed, there was a reference hidden deep into the object structure that was still being referenced by an endpoint.  Thanks for pointing out what should've been obvious.

BTW, I've been able to make facets work for fields, but not for individual enum values.  Is this supported at the moment?
